en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,627 @@
+
+This guide walks you through the following:
+
+- Developing, deploying, testing, and publishing a REST API. 
+
+- How an external application consumes the published REST API.
+
+- How to release new versions of the REST API with added functionality.
+
+Let's consider a use case where a developer named Jane designs an application for reading lists. A Choreo developer named John develops a REST API for this application to consume. The two developers will develop three versions of the solution in three iterations. Each new version will introduce new functionality as described below:
+
+
+- **Version 1**:
+
+    This version introduces the basic functionality where the application user can carry out the following activities for the reading list:
+
+     - Add books
+     - Update records of books
+     - Retrieve the complete reading list
+     - Delete books
+  
+       The information on the reading list will be removed from the system if you undeploy the REST API.
+
+- **Version 2**
+
+    This version involved connecting an external database to the REST API so that the reading list is persisted in the system memory even after the REST API is undeployed and redeployed.
+
+- **Version 3**
+
+    This version includes a front-end application that uses WSO2 Asgardeo as the identity provider. The front end-application allows different users to sign in with unique IDs and maintain separate reading lists.
+
+## Step 1: Sign-up to Choreo and create an organization
+
+First, let's sign up to Choreo and create your organization:
+
+1. In [https://wso2.com/choreo/](https://wso2.com/choreo/), click **Try Choreo Now**.
+
+2. Sign in to Choreo using your Google, GitHub, or Microsoft account.
+
+3. Enter a unique organization name. For example, `Stark Industries`.
+
+    ![Create an organization in Choreo](../assets/img/references/enterprise-login/create-choreo-organization.png){.cInlineImage-small}
+
+4. Access and read the Privacy Policy and Terms of Use via the provided links, and select the **I agree with the Privacy Policy and Terms of Use** checkbox if you agree with them.
+
+5. Click **Create** to add the new organization.
+
+6. When the **Welcome to Choreo!** message appears, click either **Decline** or **Confirm** as you prefer depending on whether you want Choreo to send you emails with product updates, resources, and event invites.
+
+You will be viewing the **Home** page of the Choreo Console.
+
+
+## Step 2: Create a REST API
+
+Let's create your first REST API by following the steps given below:
+
+!!! tip "Before you begin"
+    You need to create a GitHub repository that you own and manage to save the REST API implementation. For this guide, you can fork [https://github.com/Asitha/choreo-reading-list-application-example](https://github.com/Asitha/choreo-reading-list-application-example).<br/><br/>Install the ChoreoApp extension to this repository to enable Choreo to connect to it.
```

Comment:
We should use https://github.com/wso2/choreo-getting-started-guide-application/ repository. Things are not updated yet, but remember to update before going live with it.